db/column_family.h

```diff
@@ -457,8 +454,7 @@ class ColumnFamilyData {
   // IMPORTANT: Only call this from DBImpl::InstallSuperVersion()
   void InstallSuperVersion(SuperVersionContext* sv_context,
                            InstrumentedMutex* db_mutex,
-                           const MutableCFOptions& mutable_cf_options,
-                           bool noImmMemoryContribution = false);
```

Comment:
Remove changes made during #8454 when the mempurge process was happening in the middle of the SwitchMemtable process.